lib/shelley/src/Cardano/Wallet/Shelley/Pools.hs

```diff
@@ -172,27 +179,29 @@ newStakePoolLayer gp nl db = StakePoolLayer
 --
 --
 
--- | Stake Pool data fields that we can fetch from the node over Local State
--- Query.
-data PoolLsqMetrics = PoolLsqMetrics
+-- | Stake pool-related data that has been read from the node using a local
+--   state query.
+data PoolLsqData = PoolLsqData
```

Comment:
> Since you're changing the name anyway... I don't think `Lsq` is descriptive enough of what the data is. It only describes the mechanism used to fetch it. How about `PoolStateFromLedger`?

I do tend to agree with you here.

But on the other hand, the data for `PoolLsqData` comes (in part) from another data type called `NodePoolLsqData` (defined in `Cardano.Wallet.Shelley.Network`). See the function `combineLsqData` for more info.

If we were to change this name, I feel that we should also change the name of `NodePoolLsqData` and `combineLsqData` for consistency, as these manipulate the same sets of data.

But perhaps that is outside the scope of this PR? We could always make another PR with these changes (to avoid making this PR too big).